axum/src/json.rs

```diff
@@ -186,7 +186,7 @@ where
     T: Serialize,
 {
     fn into_response(self) -> Response {
-        let mut buf = BytesMut::new().writer();
+        let mut buf = BytesMut::with_capacity(128).writer();
```

Comment:
Maybe add a comment about why `128` specifically. Can just link to the code in `serde_json`.

Reply:
Could also use a docs.rs link: https://docs.rs/serde_json/1.0.82/src/serde_json/ser.rs.html#2189
(could be longer-lived, but who knows)

Reply:
Done!